index.html

```diff
@@ -1049,6 +1049,371 @@ <h2>Evidence</h2>
         </pre>
 
       </section>
+<section>
+      <h1>Subject Not Equal To Holder</h1>
+
+      <p>
+This section describes how the data model supports use cases in which the holder
+ of the credential is not the subject of the credential.
+      </p>
+
+      <p>
+The following diagram depicts those instances when the subject is not equal to 
+the holder.
+     </p>
+
+      <figure>
+        <img style="margin: auto; display: block;" width="75%" src="diagrams/subject-ne-holder.svg">
+        <figcaption style="text-align: center;">
+Subject Holder Relationships in Verifiable Credentials.
+        </figcaption>
+      </figure>
+
+     <p>
+The following sections describe how each of these instances are handled in 
+the data model
+     </p>
+
+
+    <section>
+        <h2>Credential Uniquely Identifies Subject</h2>
+
+     <p>
+In this case, the claim may contain multiple properties that each provide 
+an aspect of the identity of the subject
+   </p>
+
+        <pre class="example nohighlight" title="An example of a credential 
+that uniquely identifies a subject">
+
+{
+	"id": "http://dmv.example.gov/credentials/3732",
+	"type": ["Credential", "ProofOfAgeCredential"],
+	"issuer": "https://dmv.example.gov/issuers/14",
+	"issued": "2010-01-01",
+	"claim": {
+		"id": "did:example:ebfeb1f712ebc6f1c276e12ec21",
+		"name": "Mr John Doe",
+		"address": "10 Some Street, Anytown, ThisLocal, Country X",
+		"dateOfBirth": "2001-01-01",
+		"gender": "male",
+		"placeOfBirth": "Anytown, ThisLocal, Country X",
+		"nationality": "X",
+	},
+	"proof": { ...
+	}
+}
+        </pre>
+
+
+   </section>
+
+
+   
+   <section>
+      <h2>Holder Acts on Behalf of Subject</h2>
+
+      <p>
+When the holder wants to act on behalf of the subject, the holder creates 
+a Profile, in which the profile ID is the ID of the holder, and the relationship 
+property states the relationship between the holder and the subject.
```

Comment:
I agree, and I had thought of that potential security vulnerability already. It can be solved, but it places an extra burden on the verifier to validate this relationship in order to close the vulnerability. Clearly it is better if the subject can assert the relationship for the verifier, but this is not always possible e.g. when the subject is a pet, or a young child, and the holder is the pet's owner or parent. Thus we have to have a way of covering this situation as well, which is why this mechanism is proposed. So rather than remove the facility, we should add appropriate text to cover the situation.

Reply:
If it isn't always possible, how does any verifier know when or when not to enforce the relationship check?
And it doesn't seem secure at all to me to allow a situation where the verifier can't check that the subject agrees to the relationship... if Bobby is the child, and Alice is claiming to be her mother, but Bobby can't confirm or deny, what do we do in the case that Mallet is claiming to be Bobby's parent as well but is acting maliciously?

Reply:
Further down, Manu suggested that this issuer could state the relationship between holder and subject in another VC. This can also be used to solve the issue of for example, a parent speaking on behalf of their child. The issuer validates that the parent is the parent of the child, then issues the child's VC to the parent along with a second VC that states the relationship between the parent and the child.